bigbluebutton-tests/playwright/sharednotes/sharednotes.js

```diff
@@ -114,9 +123,48 @@ class SharedNotes extends MultiUsers {
     await exportButtonLocator.click();
 
     const exportPlainTextLocator = getExportPlainTextLocator(this.modPage);
-    page.waitForEvent('download');
-    await exportPlainTextLocator.click();
-    await sleep(500);
+    const exportHtmlLocator = getExportHTMLLocator(this.modPage);
+    const exportEtherpadLocator = getExportEtherpadLocator(this.modPage);
+
+    //.txt checks
+    const [download] = await Promise.all([
+      page.waitForEvent('download', { timeout: 5000 }),
+      exportPlainTextLocator.click(),
+    ]);
+    await expect(download).toBeTruthy();
+    const filePath = await download.path();
+    const content = await readFileSync(filePath, 'utf8');
+
+    const txtFileExtension = (download._suggestedFilename).split('.').pop();
+    await checkTextContent(txtFileExtension, 'txt');
+    await checkTextContent(content, e.message);
```

Comment:
use the page function `handleDownload` to manipulate this. It will be useful in other places